tests/models/test_gptq_marlin.py

```diff
@@ -94,3 +94,29 @@ def test_models(
         name_0="gptq",
         name_1="gptq_marlin",
     )
+
+
+@pytest.mark.flaky(reruns=3)
+@pytest.mark.skipif(not is_quant_method_supported("gptq_marlin"),
```

Comment:
Should also avoid calling it here, since it'll be imported by the distributed test